public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-new-site.php

```diff
@@ -176,14 +187,19 @@ public function maybe_create_new_site( $wordcamp_id ) {
 		) );
 
 		if ( is_int( $this->new_site_id ) ) {
-			update_post_meta( $wordcamp_id, '_site_id', $this->new_site_id );    // this is used in other plugins to map the `wordcamp` post to it's corresponding site
+			// This is used in other plugins to map the `wordcamp` post to it's corresponding site.
```

Comment:
```suggestion
			// `_site_id` is used in other plugins to map the `wordcamp` post to it's corresponding site.
```

`SHOULD` - Since the comment is no longer tied to the line specifically, it seems helpful to make "this" explicit.